### PR TITLE
Feature: allow to disable system proxy usage with conan.conf

### PR DESCRIPTION
**Pull request by SSE4**
closes: #5980
Changelog: Feature: allow to disable system proxy usage with conan.conf
Docs: TODO

### CODE REVIEW

File: conans/client/conf/__init__.py [hunk 1]
```diff
@@ -170,6 +170,8 @@
 # You can skip the proxy for the matching (fnmatch) urls (comma-separated)
 # no_proxy_match = *bintray.com*, https://myserver.*
 
+# use_system_proxy = True
```
Comment (SSE4) [inline conans/client/conf/__init__.py]: do we need an environment variable like `CONAN_USE_SYSTEM_PROXY`?

**Comment by SSE4** [comment]
an alternate solution could be setting `NO_PROXY` to `*`. is it a better thing?

**Comment by lasote** [comment]
Probably that (if works, because I remember all of this kind of random depending on the system) will disable also the proxy adjustments from Conan.

**Comment by SSE4** [comment]
yes, so besides `trust_env` I don't see the way how to disable system proxy. hijacking into `requests` seems to be too fragile.
what would you recommend?

**Comment by lasote** [comment]
I would recommend not working on an issue that has not been planned. :P

**Comment by lasote** [comment]
I asked @monsdar to clarify if the mechanism is working or not. The issue starts with the fact that clearing the section to deactivate the system proxies, and the issue is the `conan config install` not being able to merge the changes. But if removing the section doesn't work, the original issue makes no sense at all.

**Comment by memsharded** [comment]
Closing as outdated, also the conf system has changed to ``global.conf``, ``conan.conf`` dissapears.

**Pull request closed without merging**